### PR TITLE
Blog post announcing PR general availability

### PR DESCRIPTION
**Pull request by ericholscher**
The goal of this post is a couple things:

* Formally announce the PR builder
* Alert users to the concurrency limits involved with this change

It's pretty short, since the previous post and docs cover a lot of the detail.

### CODE REVIEW

File: pull-request-builder-general-availability.rst [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+This moves documentation into your continuous integration pipeline,
+and improves the workflow for everyone working on documentation.
+
+.. figure:: img/github-build-status-reporting.gif
+    :alt: GitHub Build Status Reporting for Pull Requests.
+    :width: 70%
+
+    GitHub Build Status Reporting for Pull Requests
+
+You can read more about this new feature in our :doc:`previous blog post announcing it </building-docs-for-pull-requests>` and in our `documentation for the feature <https://docs.readthedocs.io/en/latest/guides/autobuild-docs-for-pull-requests.html>`_.
```
Comment (stsewd) [inline pull-request-builder-general-availability.rst:24]:
```suggestion
You can read more about this new feature in our :doc:`previous blog post announcing it </building-docs-for-pull-requests>` and in our `documentation for the feature <https://docs.readthedocs.io/page/guides/autobuild-docs-for-pull-requests.html>`_.
```

File: pull-request-builder-general-availability.rst [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+
+Serving docs for every commit
```
Comment (stsewd) [inline pull-request-builder-general-availability.rst:27]: This isn't totally accurate, we only keep the docs from the latest commit of the branch, I think there was a feature request related to this, serving docs from each commit on a separate namespace.

File: pull-request-builder-general-availability.rst (hunk 1 above)
Comment (stsewd) [reply pull-request-builder-general-availability.rst:24]: I have updated the docs for pull requests here https://github.com/readthedocs/readthedocs.org/pull/7823

**Comment by ericholscher** [comment]
I think we should ship this tomorrow. @humitos you can merge if you'd like, or we can wait until I wake up and then update the feature flags together.

**Comment by humitos** [comment]
> Also, we need to remove this message from the docs:
> 
> > This feature is enabled by a feature flag currently in testing. You can reach our support at [support@readthedocs.com](mailto:support@readthedocs.com) to ask for it to be enabled.
> 
> Besides, for another/different PR, the documentation page could also mention that only the HTML version is built. A little general review of the whole page would be also good, since things have changed from when it was written.

This is done in https://github.com/readthedocs/readthedocs.org/pull/7823

**Pull request merged**